public/editor/stylesheets/editor.less

```diff
@@ -338,7 +338,6 @@
 
   img {
       width: 22px;
-      heigh: 22px;
```

Comment:
@flukeout helped me go through the lesslint errors, so I'm following his lead here.